### PR TITLE
Apply classname to button, not content.

### PR DESCRIPTION
**Pull request by korbinancell**

### CODE REVIEW

File: components/button/anchor-button.jsx [hunk 1]
```diff
@@ -19,4 +19,6 @@ AnchorButton.propTypes = {
 	...BaseButton.propTypes,
 	/** The destination the AnchorButton should link to. */
 	href: PropTypes.string,
+	/** See the docs for how to override styles properly */
```
Comment (bryanrsmith) [inline components/button/anchor-button.jsx]: Won't this comment appear *in* the docs? I think accurately describing the prop would be better than telling the reader to go away.

Comment (korbinancell) [reply components/button/anchor-button.jsx]: I think that's a direct copy-paste from react-strap and we don't even have any docs on it

**Comment by korbinancell** [comment]
Looks like the button documentation is having issues as well. I'm looking into it.

**Comment by korbinancell** [comment]
The current version of babel-react-docgen cannot handle React.forwardRef and pretty much anything that is not a class component. Unfortunately, the next version requires Babel v7 which is a task in itself. https://github.com/Faithlife/styled-ui/pull/135 includes an upgrade to v7 so I will look into it when I get the time.

**Comment by korbinancell** [comment]
@bryanrsmith this is ready for review/merge

**Pull request merged**